extensions/EXT/EXT_texture_format_BGRA8888.txt

```diff
@@ -135,12 +141,35 @@ Interactions with the OpenGL ES 2.0 specification
     and the number of bits each format contains for color (R, G, B, A),
     depth (D), and stencil (S) components:
     
-    |-----------------|------------------|------|------|------|------|------|------|
-    | Sized           | Renderable       | R    | G    | B    | A    | D    | S    |
-    | Internal Format | Type             | bits | bits | bits | bits | bits | bits |
-    |-----------------|------------------|------|------|------|------|------|------|
-    | GL_BGRA_EXT     | color-renderable | 8    | 8    | 8    | 8    |      |      |
-    |-----------------|------------------|------|------|------|------|------|------|
+    Sized           Renderable        R    G    B    A    D    S
+    Internal Format Type              bits bits bits bits bits bits
+    --------------- ----------        ---- ---- ---- ---- ---- ----
+    BGRA_EXT        color-renderable  8    8    8    8
+    BGRA8_EXT       color-renderable  8    8    8    8
+
+Interactions with the OpenGL ES 3.0 specification
+
+    Add the following entries to Table 3.2: Valid combinations of format,
+    type, and sized internalformat:
+
+    Format    Type           External Bytes per Pixel Internal Format
+    ------    ----           ------------------------ ---------------
+    BGRA_EXT  UNSIGNED_BYTE  4                        BGRA_EXT, BGRA8_EXT
+
+
+    Modify Section 3.8.3 (Texture Image Specification), p 128
+
+    (add the following required Texture and renderbuffer color formats, p 216)
+
+    - BGRA_EXT
```

Comment:
I'm not sure why BGRA8 is missing here, given that we've added it for both texture specification and renderbuffer storage?

Reply:
Yeah, that seems like a mistake here.

Also "p 216" is wrong.

Reply:
Heh yeah, I was wondering about p216, but I figured it was a reference to something that indirectly backlinks to this table, noting the required renderbuffer storage formats.

Reply:
Looking at this, I think I just kinda half-did things, and probably lost a bit of track of what I was doing. This entire section seems to just be about adding an explicit interaction with ES3, because we intend to diverge from ES2 a bit there.

So this should probably be its own preceding commit, and then the "real" sized enums should be added in this commit.

Reply:
> This entire section seems to just be about adding an explicit interaction with ES3, simply to make it clearer.

Strike that last part, the reason is because we wanted to only change this for ES3 and later...

Reply:
I generally say something like "The nnn-th paragraph of section S.P.Q" in an attempt to avoid hardwiring layout dependencies. It at least is likely to work for that specific version of the spec it's written against but later point updates may defeat it. Additional context is also sometimes helpful.